content/docs/insights/get-started/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Get started
+title_tag: Get Started with Pulumi Insights Account Discovery
+h1: Get Started with Pulumi Insights Account Discovery
+meta_desc: Learn how to set up and manage Pulumi Insights accounts to gain visibility into your cloud infrastructure.
+weight: 1
+menu:
+  insights:
+    parent: insights-home
+    identifier: insights-get-started
+    weight: 1
+---
+ 
+Pulumi Insights is an intelligent infrastructure management service that helps you discover, understand, manage, and improve your cloud infrastructure. Insights improves security, compliance, and efficiency through AI-powered asset and compliance management.
+
+Pulumi Insights give you the tools to:
+
+- **Discover**: Scan and sync all your cloud infrastructure – including resources provisioned manually or through other tools like Terraform, CloudFormation, and ARM – to bring everything under a single pane of glass. Gain complete visibility across clouds, regions, and accounts.
```

Comment:
probably worth calling out that we also scan k8s containers here and will eventually support tons of pulumi providers including sass tools like github

Reply:
added k8